vault/templates.community.office-hours.youtube.md

```diff
@@ -19,7 +19,7 @@ xx:xx Highlight: Name of Highlight
 xx:xx Q&A: Name of Subject/Question
 
 Links to All Things Dendron:
-- Discord: https://discord.com/invite/xrKTUStHNZ
+- Discord: https://link.dendron.so/discord
 - Website: https://dendron.so/
 - Blog: https://blog.dendron.so/
 - Newsletter: https://buttondown.email/dendron
```

Comment:
update to switchy link

Reply:
also twitter and github

Reply:
Updated, and completed some Luma backlog items I needed to do. Now have Dendron Events publicly viewable here:
- https://link.dendron.so/luma